cryptoki-sys/Cargo.toml

```diff
@@ -12,7 +12,7 @@ repository = "https://github.com/parallaxsecond/rust-cryptoki"
 documentation = "https://docs.rs/crate/cryptoki"
 
 [build-dependencies]
-bindgen = { version = "0.57.0", optional = true }
+bindgen = { version = "0.59.1", optional = true }
```

Comment:
Do you suspect this may cause issues with the other Parsec projects?  Would it be better for me to back this out and implement the `Debug` trait by hand?

Reply:
Not sure if I'd call it an issue, actually, since we're moving forward with the dependency, but when packaging the Parsec service we need to use the same version of `bindgen` across the board, and since we use it in quite a few crates/repos it ends up being a pain.

But the changes this version introduces seem pretty good, we might need to regenerate our bindings for the other crates too :) I remember having to tell `clippy` off for flagging some stuff that `bindgen` generated